editors/sc-ide/widgets/post_window.cpp

```diff
@@ -248,13 +252,18 @@ void PostWindow::zoomOut(int steps) { zoomFont(-steps); }
 
 void PostWindow::zoomFont(int steps) {
     QFont currentFont = font();
-    const int newSize = currentFont.pointSize() + steps;
-    if (newSize <= 0)
-        return;
+    const int newSize = GenericCodeEditor::clampFontSize(currentFont.pointSize() + steps);
     currentFont.setPointSize(newSize);
```

Comment:
This function now uses ints (consistently), but 
[void GenericCodeEditor::zoomFont(int steps)](https://github.com/supercollider/supercollider/pull/6563/files#diff-305813f383824a6e3e1172ebee7b331ad6017ad5c7d3e7ca3ac07f4d9ae11186L855)
uses floats. Not a big deal, but do we want more consistency here? Sorry for being pedantic!

Reply:
np for me being pedantic, always enjoy to go for a good solution. I hope I got it more consistent by using floats